src/api/auth.ts

```diff
@@ -407,7 +407,8 @@ export function publishPackage(
                 owners: [ username ],
                 organization: organization ? organization : '',
                 teamPermissions: permissions,
-                memberPermissions: [{ name: username, permission: 'read-write' }]
+                memberPermissions: [{ name: username, permission: 'read-write' }],
```

Comment:
Yeah, Jan's format is definitely more open and leaves us with open options in the future...